build/scripts/build.ps1

```diff
@@ -273,6 +273,8 @@ function Deploy-VsixViaTool() {
         "Vsix\VisualStudioDiagnosticsWindow\Roslyn.VisualStudio.DiagnosticsWindow.vsix",
         "Vsix\VisualStudioIntegrationTestSetup\Microsoft.VisualStudio.IntegrationTest.Setup.vsix")
 
+    Get-Process -Name "devenv"
```

Comment:
If the intent is to print then please pipe directly to `| Write-Host`. 

Current form implicitly pipes to `Write-Output`. Fine in the current position but deadly if you ever refactor this to a function.